docs/contributing/docs-and-blog-components.md

```diff
@@ -156,77 +156,24 @@ Rendered, the component looks like this:
   Connect your Gatsby site to Storage Provider for automatic deployments
 </CloudCallout>
 
```

Comment:
The ComponentModel was recently made more flexible, so I think it's worth keeping the documentation for it since we'll be able to use it in more places.